webapp/src/main/java/life/qbic/email/Email.java

```diff
@@ -0,0 +1,23 @@
+package life.qbic.email;
+
+import life.qbic.usermanagement.registration.Recipient;
+
+/**
+ * <b>short description</b>
+ *
+ * <p>detailed description</p>
+ *
+ * @since <version tag>
+ */
+public interface Email {
```

Comment:
why an interface? I dont see why we have several implementations in the future. 

I get why you did it, because of the formatting of the content. However I dont see it as the responsibility to format the content. An email just has content that should fit the MIME type. We dont need to model it as complex as the real email world is, but formatting is not an email concern. 

An email is a data carrier, but it needs some logic that prepares the content. I would outsource this logic into an own object, and make the Email a simple data structure.